src/styles/modules/magnify.scss

```diff
@@ -0,0 +1,6 @@
+.magnify > .magnify-lens {
+  border: 3px solid $frenchGray;
+  box-shadow: none;
+  cursor: crosshair;
+  background-color: $white !important;
```

Comment:
Since the library of Magnify overwrites `background-color` using the shorthand property of `background`, we have to use `!important` for it to take effect.